src/module/apps/compendium-browser/index.ts

```diff
@@ -68,48 +73,164 @@ class CompendiumBrowser extends Application {
         return game.i18n.localize("PF2E.CompendiumBrowser.Title");
     }
 
-    static override get defaultOptions(): ApplicationOptions {
-        return {
-            ...super.defaultOptions,
-            id: "compendium-browser",
-            classes: [],
-            template: "systems/pf2e/templates/compendium-browser/compendium-browser.hbs",
+    static override DEFAULT_OPTIONS: DeepPartial<ApplicationConfiguration> = {
+        ...super.DEFAULT_OPTIONS,
+        id: "compendium-browser",
+        classes: ["compendium-browser"],
+        position: {
             width: 800,
             height: 700,
-            resizable: true,
-            dragDrop: [{ dragSelector: "ul.item-list > li.item" }],
-            tabs: [
+        },
+        window: {
+            controls: [
+                {
+                    action: "openSettings",
+                    icon: "fa-solid fa-cogs",
+                    label: "PF2E.CompendiumBrowser.Settings.OpenSettings",
+                },
                 {
-                    navSelector: "nav",
-                    contentSelector: "section.content",
-                    initial: "landing-page",
+                    action: "addToRollTable",
+                    icon: "fa-solid fa-list",
+                    label: "PF2E.CompendiumBrowser.RollTable.AddLabel",
                 },
                 {
-                    navSelector: "nav[data-group=settings]",
-                    contentSelector: ".settings-container",
-                    initial: "packs",
+                    action: "createRollTable",
+                    icon: "fa-solid fa-list",
+                    label: "PF2E.CompendiumBrowser.RollTable.CreateLabel",
                 },
```

Comment:
Header controls now?

Reply:
I didn't like the floating buttons from the start but there was no room to put them anywhere else. This moves them out of the way but keeps them still accessable.